### DISCUSSION TITLE
Quarkus 2.6.1.Final released - Maintenance release

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#22517] [Quarkus Blog/Website]
# Quarkus 2.6.1.Final released - Maintenance release

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-2-6-1-final-released/

**Comment by baodingfengyun** [comment]
Hello Quarkus.

**Comment by alinayebi9080** [comment]
</p>\"<script>alert("HACK")<script/>\"

**Comment by alinayebi9080** [comment]
asd

**Comment by alinayebi9080** [comment]
&lt;script&gt;alert('Hack!')&lt;/script&gt;

**Comment by alinayebi9080** [comment]
</p>&lt;script&gt;alert('You got pranked!')&lt;/script&gt;
